docs/import_specifications.ADR.md

```diff
@@ -278,4 +289,4 @@ scientific name as returned from the taxon API.
 * Providing the scientific name vs. the taxon ID seems simpler because the machinery already 
   exists to perform the query and is part of the spec.
 * Expect these plans to change as it becomes more clear how dynamic fields will work in the
-  context of bulk upload.
\ No newline at end of file
+  context of bulk import.
```

Comment:
The updates here are to change upload to import and enforce the column count restriction? 

I think that is fine and don't have much to add.

Reply:
Also add the best effort parsing and a new error type